components/Tabs/src/TabBarView/MDCTabBarView.m

```diff
@@ -443,73 +418,91 @@ - (void)layoutSubviews {
   }
 }
 
-- (void)willMoveToSuperview:(UIView *)newSuperview {
-  [super willMoveToSuperview:newSuperview];
-  self.initialScrollDone = NO;
+- (BOOL)isJustifiedLayoutStyle {
```

Comment:
Since the getter is called several times. I think it is better to override the setter and use the self.isJustifiedLayoutStyle as a BOOL flag. But this can happen later.

Reply:
I know it's a bit more computationally heavy, but for now I do prefer that it's a calculated value. That ensures we don't forget to update it and it always reflects the current state of the layout.  I agree that later we can make some internal optimizations.